### ISSUE TITLE
MText / MHeading `font-weight="bold"` option

### ISSUE THREAD

**Issue by ashjtan** [square/maker#309]
### Feature request

More convenient ways to apply common font-weights on MText and MHeading, e.g. 
```vue
<m-text
   font-weight="bold"
>
```
vs (current)
```vue
<m-text
   font-weight="700"
>
```

### Why?

@Ryguy11 mentioned that we should use the `font-weight` prop on MText and MHeading rather than directly applying CSS classes that apply `font-weight: bold`. I would guess that people will be further discouraged from doing so because it's inconvenient and easy to forget what the font-weight for "bold" is. 

### Alternatives

Not dying for this feature; we could do without it or add a const on the user end instead if needed.

**Comment by pretzelhammer** [comment]
note: I believe this is already possible in the latest `beta` release of Maker, and should be available on `master` once [this PR](https://github.com/square/maker/pull/302) is merged in, not 100% sure when that will be though. Is this urgent? We can get it out on `master` sooner if so.

**Comment by pretzelhammer** [comment]
this has been released as part of Maker v11.0.0

I'm closing this, but feel free to re-open if v11 doesn't satisfy your needs in some way

**Comment by Ryguy11** [comment]
Thanks @pretzelhammer!
